packages/analytics-v1.1/src/constants/integrations/integration_cname.js

```diff
@@ -0,0 +1,171 @@
+import { CNameMapping as Woopra } from '@rudderstack/analytics-js-integrations/integrations/Woopra/constants';
+import { CNameMapping as RollBar } from '@rudderstack/analytics-js-integrations/integrations/RollBar/constants';
+import { CNameMapping as QuoraPixel } from '@rudderstack/analytics-js-integrations/integrations/QuoraPixel/constants';
+import { CNameMapping as June } from '@rudderstack/analytics-js-integrations/integrations/June/constants';
+import { CNameMapping as Engage } from '@rudderstack/analytics-js-integrations/integrations/Engage/constants';
+import { CNameMapping as Iterable } from '@rudderstack/analytics-js-integrations/integrations/Iterable/constants';
+import { CNameMapping as YandexMetrica } from '@rudderstack/analytics-js-integrations/integrations/YandexMetrica/constants';
+import { CNameMapping as Refiner } from '@rudderstack/analytics-js-integrations/integrations/Refiner/constants';
+import { CNameMapping as Qualaroo } from '@rudderstack/analytics-js-integrations/integrations/Qualaroo/constants';
+import { CNameMapping as Podsights } from '@rudderstack/analytics-js-integrations/integrations/Podsights/constants';
+import { CNameMapping as Axeptio } from '@rudderstack/analytics-js-integrations/integrations/Axeptio/constants';
+import { CNameMapping as SatisMeter } from '@rudderstack/analytics-js-integrations/integrations/Satismeter/constants';
+import { CNameMapping as MicrosoftClarity } from '@rudderstack/analytics-js-integrations/integrations/MicrosoftClarity/constants';
+import { CNameMapping as Sendinblue } from '@rudderstack/analytics-js-integrations/integrations/Sendinblue/constants';
+import { CNameMapping as Olark } from '@rudderstack/analytics-js-integrations/integrations/Olark/constants';
+import { CNameMapping as Lemnisk } from '@rudderstack/analytics-js-integrations/integrations/Lemnisk/constants';
+import { CNameMapping as TiktokAds } from '@rudderstack/analytics-js-integrations/integrations/TiktokAds/constants';
+import { CNameMapping as ActiveCampaign } from '@rudderstack/analytics-js-integrations/integrations/ActiveCampaign/constants';
+import { CNameMapping as Sprig } from '@rudderstack/analytics-js-integrations/integrations/Sprig/constants';
+import { CNameMapping as SpotifyPixel } from '@rudderstack/analytics-js-integrations/integrations/SpotifyPixel/constants';
+import { CNameMapping as CommandBar } from '@rudderstack/analytics-js-integrations/integrations/CommandBar/constants';
+import { CNameMapping as Ninetailed } from '@rudderstack/analytics-js-integrations/integrations/Ninetailed/constants';
+import { CNameMapping as XPixel } from '@rudderstack/analytics-js-integrations/integrations/XPixel/constants';
+import { CNameMapping as Gainsight_PX } from '@rudderstack/analytics-js-integrations/integrations/Gainsight_PX/constants';
+import { CNameMapping as Userpilot } from '@rudderstack/analytics-js-integrations/integrations/Userpilot/constants';
```

Comment:
@maheshkutty @MoumitaM , Comscore is missing in this file. Please raise a PR to the release branch.

Reply:
This has been addressed in https://github.com/rudderlabs/rudder-sdk-js/pull/2366